src/main/resources/crimson.mixins.json

```diff
@@ -3,6 +3,7 @@
   "package": "modfest.teamgreen.mixin",
   "compatibilityLevel": "JAVA_8",
   "mixins": [
+		"AccessorAxeItem",
```

Comment:
wtf

Reply:
Fixed in 500f65b0a3f45983d78cbaa9d5223dbddf1fea9b.